### PR TITLE
Rename project to Scientific Python lecture notes

### PR DESCRIPTION
**Pull request by tupui**
As discussed during today at the Scientific Python summit, I propose to rename the project to "Scientific Python lecture notes".

This removes yet another confusion in the current state of the ecosystem and clarify that this is not part of the SciPy library (which is a trademark concern on its own.)

If/once this is approved, the name of the repository should be changed accordingly. (And there is going to be the question about the web domain and where/how this website is administrated.)

cc @jarrodmillman

### CODE REVIEW

**Review by jarrodmillman — APPROVED** [review]

File: README.rst
Comment (mdhaber) [inline README.rst:11]:
```suggestion
This repository gathers some lecture notes on the scientific Python
```
We might not want to capitalize "scientific" in some places if we want to emphasize that "Scientific Python" does not claim the ecosystem.

File: pyximages/README.md
Comment (mdhaber) [inline pyximages/README.md:4]: Here, "Scientific Python Lecture Notes" is all capitalized; elsewhere, "lecture notes" is not. Although it's reasonable to say that the "lecture notes" belong to "Scientific Python", capitalizing everything makes it ambiguous whether the notes are those of "Scientific Python" or "scientific Python". We should decide one way on this and keep it consistent.

**Comment by tupui** [comment]
I fixed the casing. Capitalised when talking about the lecture notes and not otherwise.

**Review by mdhaber — APPROVED** [review]
This looks good now. Thanks!

**Comment by jarrodmillman** [comment]
@GaelVaroquaux I think we discussed doing this last summer, but I wanted to double-check with you before merging.

**Comment by GaelVaroquaux** [comment]
No problem from my side. :+1:

**Pull request merged**
